### PR TITLE
Adding canonical tags on gitcoin.co/gas

### LINKED ISSUE #2899: Gitcoin SEO Audit: Add Meta Tags to issue/bounty pages
### User Story

As a Gitcoin admin I'd like the Gitcoin site `issue/bounty` pages to have metatags (content descriptors that help tell search engines what a web page is about).

### Why Is this Needed

On the SEO side, this is high impact for Gitcoin. 

### Description

[SEO Audit Google Document, Issue Bounty Pages tab](https://docs.google.com/spreadsheets/d/1k8w8PX59XISYWaRLmUwQvdsLDCiDH0jBIY08qsKpruw/edit?usp=sharing)

The description of each `issue/bounty` page should be populated within their respective "Meta Description" and title. The first 200 characters are enough for each. 

Pull the bounty title into the `<title>` tag and 2 sentences from the bounty description into the  `<meta name="description".` tag, and make note of this.  

### Current Behavior

Title and meta tag descriptions need updating for the `issue/bounty` pages.

### Expected Behavior

Title and meta tag descriptions are in the meta tags on the actual issue page, and have been pasted into the SEO Audit Google Document for `issue/bounty` pages.

### Definition of Done

- [ ] for `issue/.*` pages (from the explorer), pull the bounty title into the meta title tag (`<title>`) if not done so already
- [x] for `issue/.*` pages (from the explorer), pull the bounty description into the meta description tag (`<meta name="description"...`) if not done so already
- [x] make note of the title and meta description phrases in the SEO Audit Googlesheet, similar to the Meta Tag Recommendations tab - edit: not required anymore.

### Additional Information

Example: https://explorer.bounties.network/bounty/1799 from bounties network (hit inspect)
Example issue/bounty page from Gitcoin: https://gitcoin.co/issue/status-im/status.im/127/1738

### PR DESCRIPTION
**Pull request by nanspro**
##### Description

Adding canonical tags in '/gas' and '/gas/history'

##### Affected core subsystem(s)

Webpages

##### Refers/Fixes
Fixes #2899

### CODE REVIEW

**Comment by nanspro** [comment]
@SaptakS i have updated the branch, could you review it again?

File: app/dashboard/templates/shared/gas_history.html [hunk 1]
```diff
@@ -15,6 +15,8 @@
   along with this program. If not, see <http://www.gnu.org/licenses/>.
 {% endcomment %}
 
+<link rel="canonical" href="https://gitcoin.co/gas/history">
```
Comment (thelostone-mc) [inline app/dashboard/templates/shared/gas_history.html]: `{% url 'gas_history_view' %}`

File: app/retail/templates/gas.html [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {% extends 'onepager/base.html' %}
 {% load i18n static %}
 {% block 'scripts' %}
+<link rel="canonical" href="https://gitcoin.co/gas">
```
Comment (thelostone-mc) [inline app/retail/templates/gas.html]: {% url 'gas' %}

**Comment by nanspro** [comment]
@thelostone-mc  updated :smile:

**Pull request merged**